ui_framework/src/components/panel/_panel.scss

```diff
@@ -20,4 +21,8 @@
   &.kuiPanel--shadow {
     @include kuiBottomShadow;
   }
+
+  &.kuiPanel--flexGrowZero {
```

Comment:
This is kind of mixing components together. Panel doesn't really need this except when it's in in an item. We've been pretty good about not having components intermingle in the SCSS.

Reply:
Let's talk some more about the FlexItem and Panel stuff over Zoom.